packages/cli-kit/src/private/node/api.ts

```diff
@@ -70,7 +70,7 @@ async function makeVerboseRequest<T extends {headers: Headers; status: number}>(
       }
       const sanitizedHeaders = sanitizedHeadersOutput(responseHeaders)
 
-      if (err.response.errors?.some((error) => error.extensions.code === '429') || err.response.status === 429) {
+      if (err.response.status === 429 || errorsInclude429(err)) {
```

Comment:
Can you help me understand how this could be the case? Is `graphql-request` (the package we get this type from) outdated?

Reply:
GraphQL returns a 401 with this `Invalid API key or access token (unrecognized login or wrong password)` error message when auth fails ([code](https://github.com/Shopify/shopify/blob/d19e4bbec348d1a4d0066e993e5c2bbce5af6ec1/areas/core/shopify/components/access_and_auth/app/controllers/concerns/login_protection.rb#L224)). So we do expect that this error can be a string.

I've updated the `GraphQLResponse` type so that `errors ` can be a string or GraphQLError[].